htcanalyze/main.py

```diff
@@ -21,15 +21,22 @@
 
 import configargparse
 
+# import own classes
 from htcanalyze.htcanalyze import HTCAnalyze, raise_value_error
 from htcanalyze.logvalidator import LogValidator
+# import for colorful output
 from rich import print as rprint, box
 from rich.progress import Table
 
-# typing identities
+
+# after analyzing, the information for each log file is saved
+# in it's on dictionary
 log_inf_list = List[dict]
+# dictionary with contents of all valid log files as a string
 list_of_logs = List[str]
+# save start of script
```

Comment:
If this the saving the start time, then we can get rid of the comment, and name the variable to: `start_time`

Reply:
It also doesn't do this. It reassigns the `datetime.datetime` object to `date_time`. Any calls to the `datetime.datetime()` function can now also be done as `date_time()`. @mathisloevenich I strongly suggest doing such redeclarations either in the import statements, i.e., `import datetime.datetime as date_time`, `from datetime import datetime as date_time`, or not doing them at all. This creates unnecessary confusion.